src/Shared/UnitTests/TestEnvironment.cs

```diff
@@ -331,6 +331,14 @@ public void WriteLine(string format, params object[] args)
                 Console.WriteLine(format, args);
             }
         }
+
+        /// <summary>
+        /// MSBuild launches the debugger on ErrorUtilities exceptions when in DEBUG. Disable this in tests that assert these exceptions.
+        /// </summary>
+        public void DoNotLaunchDebugger()
+        {
+            SetEnvironmentVariable("MSBUILDDONOTLAUNCHDEBUGGER", "1");
```

Comment:
I thought we always set this for tests. Did a failure cause a test hang?

Reply:
I had written a few tests to assert failures from ErrorUtilities.Verify* which crashed due to the debugger window launching, and setting that env var fixed them. But then I deleted those tests (because I deleted the underlying code they tested), and I decided to keep this helper because it sounds useful to have.